grc/tests/resources/test_python_bokeh_gui_workflow.grc

```diff
@@ -0,0 +1,114 @@
+options:
+  parameters:
+    alias: ''
+    author: rootkid
```

Comment:
this really startled me when I skimmed the YAML files :D nope, not a self-confessing rootkit :D